api_docgen/Format_tutorial.mld

```diff
@@ -291,6 +291,9 @@ vertical box!
 + End your main program by a [print_newline ()] call, that flushes the
 pretty-printer tables (hence the output).  (Note that the top-level loop of the
 interactive system does it as well, just before a new input.)
++ By default, a line is assumed to be 78 columns wide. To set a different width
+you can use ([Format.set_margin]) if you're using ([Format.printf]) or
```

Comment:
Using `Format.set_margin` is more often than not a mistake, it is better to call `Format.set_geometry` which require to set the maximum indentation length at the same time.

Similarly, it seems better to state `[Format.pp_set_geometry ppf ~margin ~max_indent]` sets the geometry of its `ppf` argument whereas `[Format.set_geometry]` sets the geometry of the standard output formatter.